### PR TITLE
[tests-only][full-ci] fix tests for reva

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
- removed duplicate step
  ```diff
  -Then for user :sharee the space Shares should contain these files
  +Then user :sharee should have a share :share shared by user :sharer from space :space
  +Then user :sharee should not have a share :share shared by user :sharer from space :space
  ```
- fixed tests for reva (https://drone.owncloud.com/cs3org/reva/5743/15/6)
  ```
  Syntax error (JsonException)
  ...
  Type error: FeatureContext::getJsonDecodedResponse(): Return value must be of type array, null returned (Behat\Testwork\Call\Exception\FatalThrowableError)
  ```
- skipped sharing tests for reva

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by saw-jan** [comment]
bump PR in reva: https://github.com/cs3org/reva/pull/4976

**Comment by saw-jan** [comment]
bump in reva fails: https://drone.owncloud.com/cs3org/reva/5748/11/6

File: tests/acceptance/bootstrap/SharingNgContext.php [hunk 1]
```diff
@@ -1800,24 +1779,46 @@ public function checkIfShareExists(string $share, string $sharee, string $sharer
...
+				break;
 			}
 		}
-		Assert::assertTrue($foundShareInSharedWithMe, "Share '$share' was not found in the shared-with-me list");
+		Assert::assertSame($shouldExist, $foundShareInSharedWithMe, "Share '$share' was not found in the shared-with-me list");
 	}
 
 	/**
-	 * @Then user :sharee should have a share :share shared by user :sharer
+	 * @Then user :sharee should have a share :share shared by user :sharer from space :space
```
Comment (PrajwolAmatya) [inline tests/acceptance/bootstrap/SharingNgContext.php:1797]: Is it possible to merge the step def as `(should|should not) have a share...` ?

Comment (saw-jan) [reply tests/acceptance/bootstrap/SharingNgContext.php:1797]: should be

**Review by PrajwolAmatya — APPROVED** [review]
LGTM

**Pull request merged**